test/e2e/_helpers.js

```diff
@@ -0,0 +1,230 @@
+  return option.innerText
+}
+
+/**
+ * Fill in an autocomplete field
+ *
+ * @param {object} [field]
+ * @param {Selector} [field.selector] - A TestCafe selector
+ * @param {string|number} [field.value] - Text value or index to select. If undefined, will select a random item
+ *
+ * @returns {string} - value of the selected item
+ */
+export async function fillAutocomplete({ selector, value }) {
+  await t.click(selector)
+
+  const optionsSelector = '.autocomplete__menu .autocomplete__option'
+  const autocompleteMenuOptions = await selector.parent().find(optionsSelector)
+
+  return selectOption({
+    value,
+    options: autocompleteMenuOptions,
+  })
+}
+
+/**
+ * Fill in a radio or checkbox field
+ *
+ * @param {object} [field]
+ * @param {Selector} [field.selector] - A TestCafe selector of radios or checkboxes
+ * @param {string|number} [field.value] - Text value or index to select. If undefined, will select a random item
+ *
+ * @returns {string|array} - value of the selected item or array of items selected
+ */
+export async function fillRadioOrCheckbox({ selector, value }) {
+  const options = selector
+    .parent('fieldset')
+    .find('[type="radio"] ~ label, [type="checkbox"] ~ label')
+  return selectOption({
+    value,
+    options,
+  })
+}
+
+/**
+ * Fill in a text or textarea field
+ *
+ * @param {object} [field]
+ * @param {Selector} [field.selector] - A TestCafe selector
+ * @param {string|number} [field.value] - Text value to enter
+ *
+ * @returns {string} - value of the selected item
+ */
+export async function fillTextField({ selector, value }) {
+  await t.typeText(selector, value, { replace: true })
+  return selector.value
+}
+
+/**
+ * Fill in form details on page
+ *
+ * @param {array} [fields] - An array of fields to fill in
+ * @param {Selector} [fields.selector] - TestCafe selector of field element
+ * @param {string} [fields.value] - Value to enter/select
+ * @param {string} [fields.type] - Type of field. Can be `autocomplete`, `radio`, `checkbox`
+ *
+ * @returns {object} - object of filled in values
+ */
+export async function fillInForm(fields = []) {
+  const filledInFields = {}
+
+  for (const [key, field] of Object.entries(fields)) {
+    switch (field.type) {
+      case 'autocomplete':
+        filledInFields[key] = await fillAutocomplete(field)
+        break
+      case 'radio':
+        filledInFields[key] = await fillRadioOrCheckbox(field)
+        break
+      case 'checkbox':
+        filledInFields[key] = await fillRadioOrCheckbox(field)
+        break
+      default:
+        filledInFields[key] = await fillTextField(field)
+        break
+    }
+  }
+
+  return filledInFields
+}
+
+/**
+ * Get CSV files downloaded from the app today (glob pattern matched)
+ *
+ * @returns {string[]}
+ */
+export function getCsvDownloadFilePaths() {
+  const dateStamp = format(new Date(), 'yyyy-MM-dd')
+  const globPattern = `${join(
+    homedir(),
+    'Downloads'
+  )}/Moves on*(Downloaded ${dateStamp}*.csv`
```

Comment:
Pretty funky path/filename there!

Reply:
That's true. I think this is the existing code. Haven't really looked at how the download tests work too much yet.